src/chart/generateCategoricalChart.tsx

```diff
@@ -1423,21 +1420,12 @@ export const generateCategoricalChart = ({
       };
     }
 
-    /* eslint-disable  no-underscore-dangle */
     addListener() {
       eventCenter.on(SYNC_EVENT, this.handleReceiveSyncEvent);
-
-      if (eventCenter.setMaxListeners && eventCenter._maxListeners) {
-        eventCenter.setMaxListeners(eventCenter._maxListeners + 1);
-      }
     }
 
     removeListener() {
       eventCenter.removeListener(SYNC_EVENT, this.handleReceiveSyncEvent);
-
-      if (eventCenter.setMaxListeners && eventCenter._maxListeners) {
-        eventCenter.setMaxListeners(eventCenter._maxListeners - 1);
-      }
```

Comment:
Could you provide some context as to why we can remove this?

Reply:
+1

Reply:
`setMaxListeners` is not a valid method.

Reply:
https://github.com/primus/eventemitter3/commit/315f5a0ea0b00005b1ff584a9456071e14f7bd8e
proof in the pudding, nice